composer.json

```diff
@@ -37,17 +37,19 @@
         "source": "https://github.com/MyIntervals/emogrifier"
     },
     "require": {
-        "php": "~7.0.0 || ~7.1.0 || ~7.2.0 || ~7.3.0",
+        "php": "~7.1.0 || ~7.2.0 || ~7.3.0",
         "ext-dom": "*",
         "ext-libxml": "*",
         "symfony/css-selector": "^2.8 || ^3.0 || ^4.0"
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^2.15.3",
-        "squizlabs/php_codesniffer": "^3.5.0",
-        "slevomat/coding-standard": "^5.0.4",
         "phpmd/phpmd": "^2.7.0",
-        "phpunit/phpunit": "^5.7.27"
+        "phpunit/phpunit": "^7.5",
```

Comment:
Now that we've dropped PHP 5.x support, we can at least update PHPUnit to `^6.5.14` and use the namespaced classes.  However, PHPUnit 7.x requires PHP 7.1.

Reply:
I have created #783 for the PHPUnit upgrade.

Reply:
There's an earlier commit that limited psalm to 3.2 so it could be used alongside php 7.0.

Reply:
negated by rebased sprint of psalm implementation.